releasenotes/notes/1.2/add-random-clifford-util-5358041208729988.yaml

```diff
@@ -1,7 +1,7 @@
 ---
 features_circuits:
   - |
-    Added a new function to ``qiskit.circuit.random`` that allows to generate a pseudo-random
+    Added a new function to :mod:`qiskit.circuit.random` that allows to generate a pseudo-random
```

Comment:
This isn't a user facing module we don't have a docs page for it. Also the function isn't included in the documentation currently. We should change this to be `:mod:`qiskit.circuit` and add `random_clifford_circuit` to the root module and docs.

Reply:
Ahh that's why the hyperref didn't work -- mystery solved, thanks! 🙂